src/mgmt/srv_target.c

```diff
@@ -1048,15 +1048,14 @@ tgt_create_preallocate(void *arg)
 		 * 16MB minimum per pmemobj file (SCM partition)
 		 */
 		D_ASSERT(dss_tgt_nr > 0);
+		D_ASSERT((tca->tca_scm_size / dss_tgt_nr) >= (1 << 24));
 		if (!bio_nvme_configured(SMD_DEV_TYPE_META)) {
-			rc = tgt_vos_preallocate_sequential(tca->tca_ptrec->dptr_uuid,
-							    max(tca->tca_scm_size / dss_tgt_nr,
-								1 << 24), dss_tgt_nr);
+			rc = tgt_vos_preallocate_sequential(
```

Comment:
Why do we have to allocate sequentially in MD mode rather than parallel like in PMem?

Reply:
Its the other way round, its sequential in PMEM and parallel in MD. This change was done in phase 1 to overcome the overhead fallocate on tmpfs.